pkg/ebpf/c/tracee.bpf.c

```diff
@@ -692,14 +692,13 @@ struct recent_deleted_module_map {
 
 typedef struct recent_deleted_module_map recent_deleted_module_map_t;
 
-struct recent_inserted_module_map {
-    __uint(type, BPF_MAP_TYPE_LRU_HASH);
-    __uint(max_entries, 50); // likewise for module insertion
-    __type(key, u64);
-    __type(value, kernel_new_mod_t);
-} recent_inserted_module_map SEC(".maps");
-
-typedef struct recent_inserted_module_map recent_inserted_module_map_t;
+static u64 start_scan_time_init_shown_mods = 0;
```

Comment:
I think we need to consider when using globals in the eBPF code that they may run from multiple cores.
This means that synchronizations problems are a risk.
Even if a program cannot run in parallel, there is the risk of core caches not updating in time.
I think each use of global values should be implemented using atomic operations.